base_custom_info/README.rst

```diff
@@ -26,7 +26,7 @@ expected for a given record.
 
 To define a template, you need to:
 
-* Go to *Settings > Custom Info > Templates*.
+* Go to *Custom Info > Templates*.
```

Comment:
Note- mobile ready interface is scheduled in one of our sprints, so coming soon 😄